src/SourceGenerators/Uno.UI.Tasks/LinkerHintsGenerator/LinkerHintGeneratorTask.cs

```diff
@@ -306,13 +306,16 @@ private void BuildReferences()
 
 			string RewriteReferencePath(string referencePath, string unoUIPackageBasePath, string unoRuntimeIdentifier)
 			{
+				const string SkiaRuntimeIdentifier = "Skia";
+				const string WebAssemblyRuntimeIdentifier = "WebAssembly";
+
 				var separator = Path.DirectorySeparatorChar;
 
-				return
-					(unoRuntimeIdentifier == "Skia" || unoRuntimeIdentifier == "WebAssembly") &&
-						referencePath.StartsWith(unoUIPackageBasePath) ?
-							referencePath.Replace($"lib{separator}netstandard2.0", $"uno-runtime{separator}{unoRuntimeIdentifier}") :
-								referencePath;
+				return (unoRuntimeIdentifier.Equals(SkiaRuntimeIdentifier, StringComparison.OrdinalIgnoreCase)
```

Comment:
See my comment !

Reply:
Wouldn't we want to ignore case when comparing the identifier to "Skia" or "WebAssembly"?

The rest of the changes shouldn't affect the functionality, just cleaning up the code a little